ChatGPT improvement

Henkjan1110 opened the pull request:
ChatGPT

Henkjan1110 committed:
ChatGPT improvement

ChatGPT

nickwkoning commented on data/week2/2-conditional-statements.md:
```diff
@@ -167,7 +168,7 @@ You are an adult
 
 ## More Conditionals: else if
-In the case of multiple conditionals, we use the `else if`-command. The command `else if` is like `else`, but with an additional condition. `else if` follows the `if`-condition, and there may be multiple of them.
+In cases where there are multiple conditions to be evaluated, we use the `else if` command. Similar to the `else` command, the `else if` command allows us to specify an additional condition to check. It is placed after the initial `if` condition and can be used multiple times.
```
tweede zin deugt niet helemaal met 'similar'

nickwkoning commented on data/week2/2-conditional-statements.md:
```diff
@@ -259,7 +260,7 @@ if (isLessThan) {
-The program finally prints: 1 is less than 3!
+The program will output: 1 is less than 3!
 
 <!-- ### Modulo operator
```
Misschien een idee om het modulo operator stuk terug te plaatsen? Eigenlijk best nuttig als ze dat leren

Henkjan1110 committed:
Modulo and textual adjustment

Zou zoiets goed kunnen zijn? Heb modulo weer teruggeplaatst en iets aangepast, en de 'vreemde zin'  weggehaald.
